### PR TITLE
Makes eval runner able to load flags, cleans up 

### PR DESCRIPTION
**Pull request by amj**
not ready to merge, dockerfiles need to be updated after 'amj/cuda10' branch updated.

### CODE REVIEW

File: cc/eval.cc [hunk 1]
```diff
@@ -204,6 +204,7 @@ class Evaluator {
       if (verbose) {
         MG_LOG(INFO) << absl::StreamFormat("%s Q: %0.5f", curr_player->name(),
                                            curr_player->root()->Q());
+        MG_LOG(INFO) << curr_player->name() << " plays " << move << "\n";
```
Comment (sethtroisi) [inline cc/eval.cc]: maybe also list the move number

File: cluster/README.md [hunk 2]
```diff
@@ -109,3 +112,10 @@ minimize the number of times tensorflow has to be compiled; the base image
 compiles tensorflow without bringing in any of our source files, so changes to
 our source files don't invalidate docker's caching, which would trigger a
 rebuild.
+
+
+## Creating Clusters
```
Comment (sethtroisi) [inline cluster/README.md]: ???

File: cluster/evaluator/Dockerfile-cc [hunk 3]
```diff
@@ -1,9 +1,19 @@
 ARG project
-FROM gcr.io/$project/cc-base:latest
+#FROM gcr.io/$project/cc-base:v17-testing
```
Comment (sethtroisi) [inline cluster/evaluator/Dockerfile-cc]: "testing" is a little weird of a name

File: cluster/evaluator/cc-evaluator.yaml [hunk 4]
```diff
@@ -32,8 +32,12 @@ spec:
           value: $MODEL_BLACK
         - name: JOBNAME
           value: $JOBNAME
+        - name: EVAL_FLAGS_PATH
+          value: $EVAL_FLAGS_PATH
         - name: LD_LIBRARY_PATH
-          value: /usr/local/nvidia/lib64
+          value: /usr/local/nvidia/lib64:/usr/local/nvidia/lib
+        - name: NVIDIA_REQUIRE_CUDA
+          value: cuda >=10.0,driver>=410
```
Comment (sethtroisi) [inline cluster/evaluator/cc-evaluator.yaml]: weird that one has a space and one doesn't

File: cluster/evaluator/evaluator_cc_wrapper.sh [hunk 5]
```diff
@@ -20,11 +20,33 @@
...
+cat <<EOF > flags.txt
+  --num_readouts=1000
+  --parallel_games=2
+  --value_init_penalty=2.00
+  --virtual_losses=8
+  --resign_threshold=0.70
+EOF
+
+else
+    gsutil cp ${EVAL_FLAGS_PATH} flags.txt
```
Comment (sethtroisi) [inline cluster/evaluator/evaluator_cc_wrapper.sh]: echo "Using flags from ${EVAL_FLAGS_PATH}"

File: cluster/evaluator/launch_eval.py [hunk 6]
```diff
@@ -49,27 +52,19 @@ def launch_eval_job(m1_path, m2_path, job_name, bucket_name, completions=5):
...
-    resp_bw = api_instance.create_namespaced_job('default', body=job_conf)
-
-    os.environ['MODEL_WHITE'] = m1_path
-    os.environ['MODEL_BLACK'] = m2_path
-    os.environ['JOBNAME'] = job_name + '-wb'
-    env_job_conf = os.path.expandvars(raw_job_conf)
-    job_conf = yaml.load(env_job_conf)
-    job_conf['spec']['completions'] = completions
-
-    resp_wb = api_instance.create_namespaced_job('default', body=job_conf)
```
Comment (sethtroisi) [inline cluster/evaluator/launch_eval.py]: where did this go?

Comment (amj) [reply cluster/evaluator/launch_eval.py]: default params are parallel_games=2 now :)

File: cluster/evaluator/cc-evaluator.yaml (hunk 4 above)
Comment (amj) [reply cluster/evaluator/cc-evaluator.yaml]: Fixed.

File: cluster/README.md (hunk 2 above)
Comment (amj) [reply cluster/README.md]: Reverted.

File: cluster/evaluator/Dockerfile-cc (hunk 3 above)
Comment (amj) [reply cluster/evaluator/Dockerfile-cc]: Reverted.

**Pull request merged**